tests/test_utils.py

```diff
@@ -0,0 +1,156 @@
+"""Tests utility functions."""
+
+import os
+import pandas as pd
+from pandas.testing import assert_frame_equal
+import sys
+import subprocess
+import unittest
+
+if os.environ.get("DATABRICKS_RUNTIME_VERSION", None):
+    sys.path.append("../src")
+    # If your cluster is 15.0.0 or higher, you should install
+    # requirements-test.txt at the cluster level by uploading
+    # it. This approach is for older clusters.
+    # db_version = os.environ.get("DATABRICKS_RUNTIME_VERSION")
+    # major = int(db_version.split(".")[0])
+    # if major < 15:
+    #     subprocess.call(["/bin/sh", "../sh/install-requirements.sh", "test"])
+
+    import util  # noqa
+    import constants
+
+
+class ResStockDataTestCase(unittest.TestCase):
+    """Test functionality of ResStock processing functions."""
+
+    @unittest.skipIf(
```

Comment:
Check with @SkittyWitty but I think she already has a ticket to get `spark` to run properly in github actions. Maybe put a TODO here and reference that ticket. It will be a great day when that works and we don't have to skip these tests.

Reply:
Nice! Yeah @SkittyWitty can you point me at this ticket?

Reply:
Hey @mikivee see the ticket [here](https://www.notion.so/rewiringamerica/Local-Spark-Testing-4aef885e20034c18b1a2fba6c355e82c?pvs=4).